sdk/client/Cargo.toml

```diff
@@ -20,6 +20,8 @@ bcs = "0.1.2"
 hex = "0.4.3"
 serde = { version = "1.0.124", features = ["derive"] }
 serde_json = "1.0.64"
+futures = "0.3.12"
+rand = "0.8.3"
```

Comment:
This new broadcasting client and these deps need to only be turned on when the "async" feature is requested if we have this code live in this crate. Actually to make things easier on you (not needing to worry about this config stuff) mind moving this code into the old `json-rpc-client` crate?